app/components/nextflow/samplesheet/header_component.rb

```diff
@@ -16,17 +17,27 @@ def initialize(namespace_id:, header:, property:, samples:, metadata_fields:, re
         @metadata_fields = metadata_fields
         @required_properties = required_properties
         @workflow_params = workflow_params
+        @selected = default_header
       end
 
       # rubocop:enable Metrics/ParameterLists
 
       private
 
-      def metadata_fields_for_field(field)
+      def metadata_fields_for_field
+        field = default_header
         options = @metadata_fields.include?(field) ? @metadata_fields : [field].concat(@metadata_fields)
         label = t('components.nextflow.samplesheet.header_component.default', label: field)
```

Comment:
Suggestion (blocking): can you rename this to `default_label` or something with `default` as part of the variable name to make this easier to follow in the `map` below.